test/lightning/policies/project_user_permissions_test.exs

```diff
@@ -146,14 +165,10 @@ defmodule Lightning.Policies.ProjectUserPermissionsTest do
           edit_project_description
         )a |> (&assert_can(ProjectUsers, &1, admin, project)).()
     end
-
-    test "cannot delete the project", %{project: project, admin: admin} do
-      refute ProjectUsers |> Permissions.can?(:delete_project, admin, project)
-    end
   end
 
   describe "Project users with the :owner role" do
-    test "can create workflows, create / edit / delete / run / rerun jobs, edit the project name, edit the project description, and delete the project.",
+    test "can create workflows, create / edit / delete / run / rerun jobs, edit the project name, edit the project description, and request project deletion.",
```

Comment:
is this "request project deletion" being tested?

Reply:
Done